apps/heft/src/cli/HeftToolsCommandLineParser.ts

```diff
@@ -229,6 +228,10 @@ export class HeftToolsCommandLineParser extends CommandLineParser {
     if (currentCwd !== buildFolder) {
       // Update the CWD to the project's build root. Some tools, like Jest, use process.cwd()
       this._terminal.writeVerboseLine(`CWD is "${currentCwd}". Normalizing to project build folder.`);
+      // If `process.cwd()` and `buildFolder` differ only by casing on Windows, the chdir operation will not fix the casing, which is the entire purpose of the exercise.
+      // As such, chdir to a different directory first. That directory needs to exist, so use the parent of the current directory.
+      // This will not work if the current folder is the drive root, but that is a rather exotic case.
+      process.chdir(path.resolve(__dirname, '..'));
```

Comment:
If we are trusting that `__dirname` maps to a physical folder, then the safest option is:
```suggestion
      process.chdir(__dirname);
```